src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -83,8 +83,7 @@ export const RichTextEditor = ({
         },
         onKeyDown: (event) => {
             if (event.code === 'Tab') {
-                // Forcing a blur event because of accessibility
-                forceToBlurActiveElement();
+                forceToFocusNextElement(event, !event.shiftKey);
```

Comment:
Is it now still possible to leave the rte again? I now cannot tab outside of the RTE anymore.